src/commands/feedback.js

```diff
@@ -1,6 +1,57 @@
 const { SlashCommandBuilder } = require('@discordjs/builders');
 const { MessageActionRow, MessageButton, MessageEmbed } = require('discord.js');
 
+async function collector(interaction, message, embed) {
+
+	const filterReaction = (reaction, user) => {
+		return !user.bot && (reaction.emoji.name === '🟩' || reaction.emoji.name === '🟨' || reaction.emoji.name === '🟥');
+	};
+
+	const collectorReaction = message.createReactionCollector({ filter: filterReaction });
+
+	collectorReaction.on('collect', async (reaction, user) => {
```

Comment:
```suggestion
	collectorReaction.on('collect', async (reaction, user) => {
		await message.reactions.removeAll();
```
Since it uses `.removeAll()` method in all the if's, I think it's good to just call it here and remove from the if's.